guides/schematics.md

```diff
@@ -0,0 +1,53 @@
+Angular Material comes packaged with Angular CLI schematics to make
+creating new components easy.
+
+## Install Schematics
+Schematics come packaged with Angular Material, so once you have
+installed the npm package, they will be available via the Angular CLI.
```

Comment:
I _think_ you can also do `ng add @angular/material` and it will install for you, but we can only test that once it's on npm